### PR TITLE
Add members as contacts for sponsors

### PR DESCRIPTION
**Pull request by margOnline**
Addresses issues #284 and #285

### CODE REVIEW

File: app/controllers/admin/sponsors_controller.rb [hunk 1]
```diff
@@ -16,6 +17,7 @@ def create
     @sponsor = Sponsor.new(sponsor_params)
     authorize @sponsor
     @sponsor.build_address unless @sponsor.address.present?
+    @sponsor.contacts.build
```
Comment (deniseyu) [inline app/controllers/admin/sponsors_controller.rb]: do you not need the same `unless` condition here?

File: app/decorators/sponsor_decorator.rb [hunk 2]
```diff
@@ -0,0 +1,19 @@
+class SponsorDecorator < Draper::Decorator
+  delegate_all
+
+  def contact_info
+    [ member_contact_details, contact_full_name, email ].flatten.delete_if(&:empty?).join("<br/>").html_safe
```
Comment (deniseyu) [inline app/decorators/sponsor_decorator.rb]: right before or after the delete_if method, please also add a `compact` - there have been issues with methods being called on nil object

File: app/decorators/sponsor_decorator.rb [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+  end
+
+  def member_contact_details
+    contacts.map do |member| 
+      h.link_to(member.full_name, member)
```
Comment (deniseyu) [inline app/decorators/sponsor_decorator.rb]: what is `h` referencing?

Comment (margOnline) [inline app/decorators/sponsor_decorator.rb]: h allows you to access helper methods inside a decorator

**Comment by margOnline** [comment]
I think this is good to go now

**Comment by deniseyu** [comment]
:+1: thanks @margOnline !

**Pull request merged**